Saasu.API.Client.IntegrationTests/Saasu.API.Client.IntegrationTests.csproj

```diff
@@ -11,7 +11,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Saasu.API.Client.IntegrationTests</RootNamespace>
     <AssemblyName>Saasu.API.Client.IntegrationTests</AssemblyName>
-    <TargetFrameworkVersion>v4.7.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>

```

Comment:
Why are you changing the target framework?

Reply:
My project was 4.6.1, and I couln't otherwise use the library.  Upgrading my framework version led to other issues.  I made this change before I intended the pull request.

Reply:
PS. It's not a downgrade.  Nuget packages should target the _lowest_ possible framework that supports it, so it can be used as widely as possible.  Packages targeted for a lower version framework can be used in a project using a higher version framework, but packages with a higher version framework can't be used in a project with a lower version.